LumberjackMain.py

```diff
@@ -122,7 +121,6 @@ def prepare_batch(replay_buffer):
     reward = torch.tensor([x[3] for x in batch_data], dtype=torch.float)
     print(".", end = "")
     done = torch.tensor([x[4] for x in batch_data], dtype=torch.float)
-    print("Done")

```

Comment:
I like having it print none so I know when it's done batching and starts training